src/targeting.js

```diff
@@ -10,6 +10,8 @@ import { getLastLocation } from './marfeelTools';
 const utils = require('./utils.js');
 var CONSTANTS = require('./constants.json');
 
+export const bidsByReferrer = {};
```

Comment:
Do we need to make this const exportable?

Reply:
solved